src/components/user-profile.component.js

```diff
@@ -144,7 +144,15 @@ export const UserProfile = ({
       </TouchableOpacity>
 
       {type !== 'org' &&
-        <TouchableOpacity style={styles.unit}>
+        <TouchableOpacity
+          style={styles.unit}
+          onPress={() =>
+            navigation.navigate('StarredRepositoryList', {
+              title: 'Starred Repositories',
+              user,
+              repoCount: starCount > 15 ? 15 : starCount,
```

Comment:
15 should be moved to a const

Reply:
@machour I noticed that there are a few not const numbers in this file such as _15_, _10_ etc. Maybe I could set all of them as const like **maxPublicRepos**, **maxStarCount**, **maxFollowers** etc. What do you think?